cinema/urls.py

```diff
@@ -1,10 +1,41 @@
-from django.urls import path
+from django.urls import path, include
+from rest_framework import routers
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import (
+    GenreList,
+    GenreDetail,
+    ActorList,
+    ActorDetail,
+    CinemaHallViewSet,
+    MovieViewSet
+)
+
+cinema_hall_list = CinemaHallViewSet.as_view(actions={
+    "get": "list",
+    "post": "create",
+})
+
+cinema_hall_detail = CinemaHallViewSet.as_view(actions={
+    "get": "retrieve",
+    "put": "update",
+    "patch": "partial_update",
+    "delete": "destroy"
+})
+
+router = routers.DefaultRouter()
+router.register("movies",
+                MovieViewSet)
```

Comment:
```suggestion
                MovieViewSet)
```
you can move MovieViewSet on the line above, it will be better a bit